setup.py

```diff
@@ -16,9 +16,9 @@
     project_urls={
         "Source": "https://github.com/algorand/py-algorand-sdk",
     },
-    install_requires=["pynacl", "pycryptodomex>=3.6.0", "msgpack"],
+    install_requires=["pynacl", "pycryptodomex>=3.6.0", "msgpack", "black"],
```

Comment:
Why did you revert your changes to the `requirements.txt` file and place black here instead? I believe this is the list of runtime dependencies, and if black is only needed during development (like most linters) `requirements.txt` is a better place for it.

Reply:
Ah okay, I will put it back to `requirements.txt`.